CONTRIBUTING.md

```diff
@@ -191,6 +191,13 @@ Improvements must have the title "[Area or module] : [Added/Refactored/Improved/
 
 New collectors must have the title "[New collector] : [Name of collector]".
 
+#### Commit messages when PRs are merged
+
+When a PR is squashed and merged into master the title of the commit message (first line) must be the PR title
```

Comment:
```suggestion
When a PR gets squashed and merged into master, the title of the commit message (first line) must be the PR title
```

Reply:
Why "gets" instead of "is" - it does not seem to be grammatically correct? (I'm curious as it could be an American / British difference that I have not seen before).